scipy/linalg/flapack_gen.pyf.src

```diff
@@ -221,6 +221,22 @@ subroutine <prefix>getrs(n,nrhs,lu,piv,b,info,trans)
 
 end subroutine <prefix>getrs
 
+subroutine <prefix>getc2(n,a,ipiv,jpiv,info)
+    ! lu,ipiv,jpiv,info = getc2(a,overwrite_a=0)
+    ! Compute an LU factorization of with complete pivoting of a general n-by-n matrix.
+    ! A = P * L * U * Q
+    threadsafe
+    callstatement {int i;(*f2py_func)(&n,a,&n,ipiv,jpiv,&info);for(i=0;i\<n;--ipiv[i],--jpiv[i++]);}
```

Comment:
This is supposed to convert from 1- to 0- based indexing for both ipiv and jpiv. Is the way I wrote it legal/correct?

Reply:
Appears to be!

Reply:
I'm a bit worried about this ipiv index corrections because if you later want to pass it back to another lower routine you need again a correction to get to the 1 indexing.

Maybe leaving as is a better option?

Reply:
`getrf` corrects to 0-indexing. Would you change that, too, or make the two wrappers inconsistent? 
I agree that it's inefficient to convert back and forth, but I think the choice was already made in `getrf`.